src/main/java/frc/trigon/robot/misc/objectdetectioncamera/SimulationObjectDetectionCameraIO.java

```diff
@@ -107,13 +108,11 @@ private Pose3d[] toPosesArray(List<Translation2d> translationsList) {
     }
 
     private boolean isEjecting() {
-        //todo: check if the robot is ejecting
-        return false;
+        return RobotContainer.INTAKE.getTargetState() != IntakeConstants.IntakeState.COLLECTING;
     }
```

Comment:
Right now every state which isn't collection is ejection, but it might not stay that way.